#!/usr/bin/env python
# MapReader.cfg
PACKAGE = "pmadmu_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("resolution", double_t, 0, "Meters per Grid Cell", 0.6, 0.0, 5.0)
gen.add("show_debug_images", bool_t, 0, "Enable for more Debugging Infos. Disable for real world application", False)
gen.add("show_debug_prints", bool_t, 0, "Enable for more Debugging Infos. Disable for Performance", False)


exit(gen.generate(PACKAGE, "map_reader", "MapReader"))